dinner_party2.rb

```diff
@@ -0,0 +1,62 @@
+# Chris Hintz
+
+puts "Welcome the the Hintz Deluxe Dinner Party Planning App!"
+
+people =[]
+
+puts "Which people will be attending this shin-dig? (please enter names).  Enter 'next' once all names have been entered"
+
+name = ""
+while name != "next"
+  name = gets.chomp.to_s
+  people.push(name)
+  if name == "next"
```

Comment:
You could put the `.push` inside of an `else` or something. Using `pop` to remove the last element is a little more work and you could end up with bugs much easier. Something like:

```rb
if name == "next"
  puts "The people going to your party are #{people}.  Sounds fun!"
else
  people.push(name)
end
```